config/webpack/webpack.config.test.js

```diff
@@ -28,16 +31,25 @@ module.exports = {
     }
   }),
   // enzyme webpack issue https://github.com/airbnb/enzyme/issues/47
+  // Please note that externals may have to change for versions of React
+  // other than 0.14.x
   externals: {
-    "cheerio": "window",
     "react/lib/ExecutionEnvironment": true,
-    "react/lib/ReactContext": true
+    "react/lib/ReactContext": true,
+    "text-encoding": "window"
   },
   module: _.assign({}, prodCfg.module, {
     // enzyme webpack issue https://github.com/airbnb/enzyme/issues/47
     noParse: [
       /\/sinon\.js/
-    ]
+    ],
+    // enzyme webpack issue https://github.com/airbnb/enzyme/issues/47
+    loaders: (prodCfg.module.loaders || []).concat([
+      {
+        test: /\.json$/,
+        loader: "json"
```

Comment:
For builder, all loaders need full path. And here, need to use module pattern. So, altogether:

```js
loader: archDevRequire.resolve("json-loader")
```

Because you already have `archDevRequire` available. And you need the _full_ name of the package (e.g, `json-loader`) which webpack just automagically normally figures out.